wp-includes/class-wp-service-workers.php

```diff
@@ -152,7 +152,7 @@ public function serve_request() {
 		$file_hash = md5( $output );
 		@header( "ETag: $file_hash" ); // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged, WordPress.PHP.NoSilencedErrors.Discouraged
```

Comment:
Just noticed this needs to be quoted per [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag#Directives):

```suggestion
		$etag      = sprintf( '"%s"', $file_hash );
		@header( "ETag: $etag" ); // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged, WordPress.PHP.NoSilencedErrors.Discouraged
```